sessions/js-math-and-numbers/calculator/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+		<title>Calculator</title>
+		<link rel="stylesheet" href="./css/styles.css" />
+	</head>
+	<body>
+		<section class="wrapper">
+			<p class="result" data-js="result">0</p>
```

Comment:
I think the `output` HTML element would be a bit better in this case: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/output
```suggestion
			<output class="result" data-js="result">0</output>
```